src/moonstone-samples/lib/DataGridListSample.js

```diff
@@ -110,12 +147,72 @@ module.exports = kind({
 		this.modelIndex = idx;
 		return records;
 	},
+	generateDataGridList: function(itemType) {
+		if (this.$.gridList) {
+			this.$.gridList.destroy();
+		}
+
+		var moreProps = {};
+
+		switch (itemType) {
+		case 'HorizontalGridListImageItem':
+			moreProps = {minWidth: 600, minHeight: 100,
+				components: [{
+					kind: HorizontalGridListImageItem,
+					bindings: [
+						{from: 'model.text', to: 'caption'},
+						{from: 'model.subText', to: 'subCaption'},
+						{from: 'model.url', to: 'source'},
+						{from: 'model.selected', to: 'selected', oneWay: false}
+					]
+				}]
+			};
+			break;
+		case 'HorizontalGridListItem':
+			moreProps = {minWidth: 600, minHeight: 100,
+				components: [{
+					kind: HorizontalGridListItem,
+					bindings: [
+						{from: 'model.text', to: 'caption'},
+						{from: 'model.subText', to: 'subCaption'},
+						{from: 'model.selected', to: 'selected', oneWay: false}
+					]
+				}]
+			};
+			break;
+		default:
```

Comment:
It has some code but I removed it. I forgot to remove default. Let me make it. Thanks. :+1: